src/components/HiddenTaskList/HiddenTaskList.jsx

```diff
@@ -0,0 +1,89 @@
+const React = require('react')
+const { connect } = require('react-redux')
+
+const HiddenTaskListItem = require('../HiddenTaskListItem')
+const hookUpStickyNav = require('../hookUpStickyNav')
+const TaskVisibility = require('../../models/TaskVisibility')
+
+class HiddenTaskList extends React.Component {
+  onRestoreClick(event) {
+    event.currentTarget.blur() // defocus button
+    this.props.dispatch({ type: 'TASKS_RESTORE' })
+  }
+
+  cancel(event) {
+    event.preventDefault()
+    this.props.cancel()
+  }
+
+  navigation(tasks) {
+    if (tasks.length < 1) {
+      return
+    }
+    return (
+      <nav className="controls-container has-text-right">
+        <label className="label">With selected:</label>
+        <button
+          type="button"
+          onClick={e => this.onRestoreClick(e)}
+          className="control button is-link"
+          id="restore-button"
+          title="Restore selected"
+        >↩️</button>
+      </nav>
+    )
+  }
+
+  emptyListMessage(tasks) {
+    if (tasks.length > 0) {
+      return
+    }
+    return (
+      <p className="content">
+        There are no issues or pull requests that have been snoozed, archived,
+        or ignored for this filter.
+      </p>
+    )
+  }
+
+  render() {
+    const { activeFilter } = this.props
+    const hiddenTasks = this.props.tasks.filter(task => TaskVisibility.isHiddenTask(task))
+    return (
+      <div>
+        <nav id="hidden-task-list-navigation" className="top-nav nav">
+          <div className="nav-left">
+            <h1 className="title">
+              <a href="#" onClick={event => this.cancel(event)}>Tasks</a>
+              <span> / </span>
+              Hidden
+              <span className="subtitle"> in &ldquo;{activeFilter}&rdquo;</span>
```

Comment:
> Any thoughts on making the hidden task list show all hidden tasks, rather than filtered?

I agree that would be preferable. I didn't do that for a technical limitation right now: we don't know the details of these hidden tasks until we go fetching them from the GitHub API. So we have to switch to a filter to make the API call to get all the issues and PRs, and then we split those between the ones that are visible and the ones that are hidden. The only data we have about hidden tasks is their type (pull or issue) and their ID. Try running `cat /Users/your_user_name/Library/Application\ Support/Electron/config.json` locally to see what's getting stored in our JSON storage file.